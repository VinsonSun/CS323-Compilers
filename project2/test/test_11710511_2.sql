int test2() {
   int a = max(1, 2);
}
