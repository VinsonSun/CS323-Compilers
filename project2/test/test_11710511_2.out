Error type 2 at Line 2: undefined function: max
